release-notes.md

```diff
@@ -1,5 +1,10 @@
 # Release 2.1.7
 
+## What's New
+* TSDK updated to 0.20.2 / CSDK 0.30.1
+
+# Release 2.1.7
```

Comment:
looks like 2.1.7 was already teed up ? probably need to just update the line 18 below?

Reply:
nevermind - instead the 2.1.7 on line 1 looks like it should be 2.1.8. I also used to copy all the headings just confirming nothing changed etc like 2.1.6
```
# Release 2.1.6
## What's New
* Automatic updates are now able to be disabled entirely. The user will still be notified updates exist
## Other changes:
*
## Bugs fixed:
*
## Dependency Updates
* TSDK updated to 0.19.9 / CSDK 0.29.4
* ```